src/libstd/sys/redox/process.rs

```diff
@@ -218,7 +218,10 @@ impl Command {
         }
 
         match self.setup_io(default, true) {
-            Ok((_, theirs)) => unsafe { self.do_exec(theirs) },
+            Ok((_, theirs)) => unsafe {
+                let Err(e) = self.do_exec(theirs);
+                e
```

Comment:
I first did `self.do_exec(theirs).err().unwrap()`. Is there a strong opinion about this? The neat one about this is static assertion